autoPlay.user.js

```diff
@@ -340,6 +347,66 @@ function firstRun() {
 	isPastFirstRun = true;
 }
 
+function purchaseBadgeItems() {
+  // Spends badge points (BP's) when joining a new game.
+  // Dict contains the priority in terms of amount to buy (percentage of purchase). Probably a nicer way to do this...
+  // Priorities based on http://steamcommunity.com/groups/MSG2015/discussions/0/598198356167623323/
+  var abilityItemPriority = [
+    [ABILITIES.WORMHOLE, 100],
+    [ABILITIES.CRIT, 0],
+    [ABILITIES.TREASURE, 0],
```

Comment:
Probably want to put 1 for Crit and treasure, otherwise `maxSpend` will always be 0 and the script will only buy wormholes and Pumped_up as fallback